### DISCUSSION TITLE
Can't use arrow_flight_sql_port

### DISCUSSION THREAD

**Discussion by ChunHou20c** [apache/doris#30271] [A - General / Q&A] [answered]
I see the option to set arrow_flight_sql_port in fe.conf, I had added the line arrow_flight_sql_port = 9040 in my fe.conf and restarted fe, however I can't seems to make fe to listen for arrow flight sql. Is this feature removed or not available? I tried the binary install of latest version 2.0.3 and stable release 1.2.7 both of them doesn't use the port.

Checking the frontend also don't have this option
![2024-01-23T16:38:39,839686117+08:00-edited](https://github.com/apache/doris/assets/93393878/de81d7ce-c32d-4019-9541-a97bc267882f)

Thanks in advance for any help

**Comment by zy-kkk** [comment, marked as answer]
This feature will be released in version 2.1
